### ISSUE TITLE
customise key and cert names

### ISSUE THREAD

**Issue by kishaningithub** [FiloSottile/mkcert#72]
Ability to customise the names of output key and cert files would be a great usability feature 

I suggest flags like 
- `--o-key-file`
- `--o-cert-file`

So 
```
mkcert example.com '*.example.org' myapp.dev localhost 127.0.0.1 ::1  --o-key-file my-cert.pem  --o-cert-file my-key.pem
```

would generate `my-cert.pem` and `my-key.pem` instead of the random looking `./example.com+5.pem` `./example.com+5-key.pem`

**Comment by liamdawson** [comment]
Alternatively, maybe the `openssl req` syntax?

```
-out outfile        Output file
-keyout outfile     File to send the key to
```

Either way, the feature would be greatly appreciated! (I might look into doing it, but I know nothing about Golang yet)

**Comment by adamdecaf** [comment]
I'd suggest avoiding adopting flag naming conventions  from `openssl`.

`--cert-file` and `--key-file` ?

**Comment by sleagon** [comment]
I just add a PR to fix this, #77 .  You should notice that, you should put the params before arguments insead of after them.

**This will not work.**
```bash
mkcert example.com '*.example.org' myapp.dev localhost 127.0.0.1 ::1  --o-key-file my-cert.pem  --o-cert-file my-key.pem
```

**This works.**

```bash
mkcert --key-file my-cert.pem  --cert-file my-key.pem example.com '*.example.org' myapp.dev localhost 127.0.0.1 ::1
```

**Comment by sleagon** [comment]
@kishaningithub @adamdecaf  It will be nice if you guys could just try this PR.

**Comment by FiloSottile** [comment]
Not a fan of adding flags, but giving in to this after seeing [scripts reimplement the naming logic to find the certificates after they are generated](https://github.com/Shopify/slate/wiki/4.-Create-a-self-signed-SSL-certificate).

**Comment by FiloSottile** [comment]
Fixed in #77
